examples/generative/corrdiff/shell_scripts/example_patch_train.sh

```diff
@@ -0,0 +1 @@
+RUN_CMD="python3 -u train.py patch_shape_x=64 patch_shape_y=64 patch_num=10 batch=2560 batch_gpu=20 precond=resloss arch=ddpmpp-cwb outdir="./output_test""
```

Comment:
Please note that config file names have changed and you need to pass `--config-name`. See the README here: https://github.com/NVIDIA/modulus/blob/main/examples/generative/corrdiff/README.md